lib/jamaica/games/number_balance_game.dart

```diff
@@ -48,31 +60,234 @@ class _NumberBalanceGameState extends State<NumberBalanceGame> {
+              operand1 != null
+                  ? DragTarget<String>(
+                      key: Key('lbox1'),
+                      builder: (context, candidateData, rejectedData) =>
+                          Container(
+                            height: 70.0,
+                            width: 70.0,
+                            decoration: BoxDecoration(
+                                color: Colors.blue,
+                                borderRadius:
+                                    BorderRadius.all(Radius.circular(16.0))),
+                            child: Center(
+                              child: Text(
+                                "$operand1",
+                                style: TextStyle(
+                                    color: Colors.white,
+                                    fontSize: 50.0,
+                                    fontWeight: FontWeight.bold),
+                              ),
+                            ),
+                          ),
+                      onWillAccept: (data) => true,
+                      onAccept: (data) => setState(() {
+                            _leftOperand1 = data;
+                          }),
+                    )
+                  : Container(),
+              op != null
+                  ? Text(
+                      "$op",
+                      style: TextStyle(
+                          color: Colors.grey,
+                          fontSize: 50.0,
+                          fontWeight: FontWeight.bold),
+                    )
+                  : Container(),
+              operand2 != null
+                  ? DragTarget<String>(
+                      key: Key('rbox2'),
+                      builder: (context, candidateData, rejectedData) =>
+                          Container(
+                            height: 70.0,
+                            width: 70.0,
+                            decoration: BoxDecoration(
+                                color: Colors.blue,
+                                borderRadius:
+                                    BorderRadius.all(Radius.circular(16.0))),
+                            child: Center(
+                              child: Text(
+                                "$operand2",
+                                style: TextStyle(
+                                    color: Colors.white,
+                                    fontSize: 50.0,
+                                    fontWeight: FontWeight.bold),
+                              ),
+                            ),
+                          ),
+                      onWillAccept: (data) => true,
+                      onAccept: (data) {
+                        setState(() {
+                          _leftOperand2 = data;
+                        });
+                      })
+                  : Container(),
+            ],
+          )),
+    );
+  }
+
+  Widget _rightExpressionLayout(String operand1, String op, String operand2) {
```

Comment:
what is difference in code between leftExpressionLayout and rightExpressionLayout?

Reply:
Sir, requested changes done.